Firestore/Source/Model/FSTFieldValue.mm

```diff
@@ -630,7 +555,7 @@ - (id)value {
                                        nanoseconds:timestamp.nanoseconds()];
     }
     case FieldValue::Type::ServerTimestamp:
-      HARD_FAIL("TODO(rsgowman): implement");
+      return [NSNull null];
```

Comment:
Perhaps add a comment to explain this behavior?

Reply:
Done.